lerna.json

```diff
@@ -1,6 +1,8 @@
 {
   "packages": [
-    "packages/spl-lending"
+    "packages/spl-lending",
+    "packages/indexer-generator",
+    "packages/marinade_finance"
```

Comment:
maybe we should now turn it into "include all packages in packages/", as this might confuse people when generating new indexers and trying to run them